src/main/java/com/greenfoxacademy/goddesstribesbackend/models/entities/ActiveBuilding.java

```diff
@@ -0,0 +1,25 @@
+package com.greenfoxacademy.goddesstribesbackend.models.entities;
+
+import javax.persistence.Entity;
+import java.time.LocalDateTime;
+
+@Entity
+public abstract class ActiveBuilding extends Building {
```

Comment:
If you did want to separate this because of the Townhall, the townhall still can have these two properties, just fill it up on creation with the same date. 
Sometimes there is no point in separating two classes, because it increases complexity too much. Just go with the simpler version instead.